static/css/layout/nav-menu.css

```
@@ -1,137 +1,150 @@
   top: 0;
   left: 0;
-  overflow-x: hidden;
+  overflow: hidden;
   display: flex;
   flex-direction: column;
-  transition: width 0.3s ease-in-out;
-  background: var(--bg);
+  transition: var(--nav-transition);
+  backdrop-filter: blur(30px);
+  -webkit-backdrop-filter: blur(30px);
+  border-right: var(--nav-border);
```

Stylelint (17.14.0)

[error] Duplicate property "border-right" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)